dotCMS/src/main/java/com/dotcms/experiments/business/result/ExperimentAnalyzerUtil.java

```diff
@@ -28,6 +40,8 @@ public enum ExperimentAnalyzerUtil {
 
     INSTANCE;
 
```

Comment:
This looks such as WebAPI

Reply:
Not really, this really analyze the Experiments Events to calculate the results, so really it is connect to CubeJs. nothing to do with a request so I think is not really a WebApI class

Reply:
Not really, this really analyze the Experiments Events to calculate the results, so really it is connect to CubeJs. nothing to do with a request so I think is not really a WebApI class